.github/workflows/bazel.yml

```diff
@@ -62,9 +62,7 @@ jobs:
         shell: bash
       - name: Yams tests
         if: always()
-        run: |
-             build_workspace_directory=$(bazelisk info workspace)
-             CC=clang bazelisk test //Tests:YamsTests --action_env=PATH --test_env=BUILD_WORKSPACE_DIRECTORY=$build_workspace_directory
```

Comment:
I might need to maintain the `PATH` thing here (dependent on github actions), but if we do I think we can move it to the .bazelrc

Reply:
Looks like it’s still needed.

Reply:
I think there's a different issue with the fixture discovery change I made, not sure why it works for me locally, I'll have to debug some more